<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PDFWidget</class>
 <widget class="QWidget" name="PDFWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">MSDOS</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonReload">
       <property name="text">
        <string>Reload</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonPrev">
       <property name="text">
        <string notr="true">&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonNext">
       <property name="text">
        <string notr="true">&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonHex">
       <property name="text">
        <string>Hex</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStrings">
       <property name="text">
        <string>Strings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMemoryMap">
       <property name="text">
        <string>Memory map</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonEntropy">
       <property name="text">
        <string>Entropy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonHeuristicScan">
       <property name="text">
        <string>Heuristic scan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxReadonly">
       <property name="text">
        <string>Readonly</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="treeWidgetNavi">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
     <widget class="QStackedWidget" name="stackedWidgetInfo">
     <widget class="QWidget" name="page_Info">
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="XFileInfoWidget" name="widgetInfo" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_Visualization">
       <layout class="QVBoxLayout" name="verticalLayout_19">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="XVisualizationWidget" name="widgetVisualization" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_VirusTotal">
       <layout class="QVBoxLayout" name="verticalLayout_16_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="XVirusTotalWidget" name="widgetVirusTotal" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_HEX">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="XHexViewWidget" name="widgetHex" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_Hash">
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="XHashWidget" name="widgetHash" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_Strings">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="SearchStringsWidget" name="widgetStrings" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_Signatures">
       <layout class="QVBoxLayout" name="verticalLayout_20DD">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="SearchSignaturesWidget" name="widgetSignatures" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageMemoryMap">
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="XMemoryMapWidget" name="widgetMemoryMap" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_Entropy">
       <layout class="QVBoxLayout" name="verticalLayout_17">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="XEntropyWidget" name="widgetEntropy" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_HeuristicScan">
       <layout class="QVBoxLayout" name="verticalLayout_18">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="NFDWidgetAdvanced" name="widgetHeuristicScan" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_Extractor">
       <layout class="QVBoxLayout" name="verticalLayout_26XX">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="XExtractorWidget" name="widgetExtractor" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_Search">
       <layout class="QVBoxLayout" name="verticalLayout_63">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="SearchValuesWidget" name="widgetSearch" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_DIEScan">
       <layout class="QVBoxLayout" name="verticalLayout_5dd">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="DIEWidgetAdvanced" name="widgetDIEScan" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_YARAScan">
       <layout class="QVBoxLayout" name="verticalLayout_6dd">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="YARAWidgetAdvanced" name="widgetYARAScan" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XFileInfoWidget</class>
   <extends>QWidget</extends>
   <header>xfileinfowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SearchStringsWidget</class>
   <extends>QWidget</extends>
   <header>searchstringswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ToolsWidget</class>
   <extends>QWidget</extends>
   <header>toolswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XEntropyWidget</class>
   <extends>QWidget</extends>
   <header>xentropywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NFDWidgetAdvanced</class>
   <extends>QWidget</extends>
   <header>nfdwidgetadvanced.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XMemoryMapWidget</class>
   <extends>QWidget</extends>
   <header>xmemorymapwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XHexViewWidget</class>
   <extends>QWidget</extends>
   <header>xhexviewwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XHashWidget</class>
   <extends>QWidget</extends>
   <header>xhashwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XVirusTotalWidget</class>
   <extends>QWidget</extends>
   <header>xvirustotalwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XExtractorWidget</class>
   <extends>QWidget</extends>
   <header>xextractorwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SearchValuesWidget</class>
   <extends>QWidget</extends>
   <header>searchvalueswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XVisualizationWidget</class>
   <extends>QWidget</extends>
   <header>xvisualizationwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SearchSignaturesWidget</class>
   <extends>QWidget</extends>
   <header>searchsignatureswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DIEWidgetAdvanced</class>
   <extends>QWidget</extends>
   <header>diewidgetadvanced.h</header>
  </customwidget>
  <customwidget>
   <class>YARAWidgetAdvanced</class>
   <extends>QWidget</extends>
   <header>yarawidgetadvanced.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
